code/src/cljs/sixsq/nuvla/ui/main/effects.cljs

```diff
@@ -24,3 +24,22 @@
         (stripejs/loadStripe)
         (.then (fn [stripe]
                  (callback stripe))))))
+
+
+(defn- before-unload-handler [event]
+  (set! (.-returnValue event) "")
```

Comment:
When closing browser tab we have a different modal than the existing one. Is it possible to have the same one for all the prompt?

Reply:
My short research didn't bring anything up for changing the native modal to a custom built one: Even changing the displayed message was deprecated in all browsers. I found one Code Pen saying it implements that but it only showed in app navigation. I try running that code locally to test it out.

We could change the in-app warn modal to using the native confirm modal but I don't know how much work that would be.

Reply:
The mentioned code pen is also not able to use a custom modal for closing the browser tab, you can test it here if you want: https://uusl0.csb.app/input
1. Change input
2. Click Home -> Custom modal -> Click No
3. Try to close the tab -> Browser native confirm dialog

From my point of view the different modals are okay: A customized, richer modal when we navigate inside the app and a clear browser alert when trying to close the app completely.

(but perhaps I'm trying to justify it retroactively)

Reply:
If it's not an easy fix, we can forget about it :)